performance_regression_test.py

```diff
@@ -277,6 +277,75 @@ def test_uniform_counter_update_bench(self):
         self.display_results(results, test_name='test_read_bench')
         self.check_regression()
 
+    def test_mv(self):
+        """
+        Test steps:
+
+        1. Run WRITE workload without materialized view
+        2. Run WRITE workload with materialized view one or more times
+        """
+        import yaml
+        def run_workload(stress_cmd, user_profile):
+            self.log.debug('Run stress test with user profile {}'.format(user_profile))
+            assert os.path.exists(user_profile), 'File not found: {}'.format(user_profile)
+            self.log.debug('Stress cmd: {}'.format(stress_cmd))
+            stress_queue = self.run_stress_thread(stress_cmd=stress_cmd, stress_num=1, profile=user_profile)
+            self.get_stress_results(queue=stress_queue)
+            self.update_test_details()
+            self.check_regression()
```

Comment:
Because you'll run check_regression twice or more.
Please make sure that the email sent by check_regression contains a valid comparison between each run.